src/scripts/main.js

```diff
@@ -22,7 +22,20 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const localList = document.createElement('ul');
```

Comment:
```suggestion
  const list = document.createElement('ul');
```
list или currentList что в этом контексте означает local?